tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Properties;
+
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+
+/**
+ * @extends RuleTestCase<PropertiesInInterfaceRule>
+ */
+class PropertiesInInterfaceRuleTest extends RuleTestCase
+{
+
+	protected function getRule(): Rule
+	{
+		return new PropertiesInInterfaceRule();
+	}
+
+	public function testRule(): void
+	{
+		$this->analyse([
+			__DIR__ . '/data/properties-in-interface.php',
+			__DIR__ . '/data/bug-8915.php',
```

Comment:
Same point here - separate test method pleaase :) (And FYI personally I wouldn't add this file here at all, the rule is straightforward and the test does not really test anything different :))

Reply:
Thanks for the pointers! I retained the first test.